docs/resources/cloud_gateway_transit_gateway.md

```diff
@@ -103,6 +107,9 @@ Requires replacement if changed. (see [below for nested schema](#nestedatt--aws_
 Required:
 
 - `kind` (String) must be "aws-transit-gateway-attachment"; Requires replacement if changed.
+- `peer_account_id` (String) Requires replacement if changed.
+- `peer_vpc_id` (String) Requires replacement if changed.
+- `peer_vpc_region` (String) Requires replacement if changed.
```

Comment:
This is unexpected and would be a breaking change. I think it's due to a duplicated `title` in the CGW spec. Try applying the following change and regenerating:

```diff
diff --git a/src/konnect/definitions/cloud-gateways/v2/openapi.yaml b/src/konnect/definitions/cloud-gateways/v2/openapi.yaml
index 0029d6c2..a4a4b13c 100644
--- a/src/konnect/definitions/cloud-gateways/v2/openapi.yaml
+++ b/src/konnect/definitions/cloud-gateways/v2/openapi.yaml
@@ -2180,7 +2180,7 @@ components:
       x-flatten-allOf: true
       allOf:
         - $ref: "#/components/schemas/BaseTransitGateway"
-        - title: AWS Transit Gateway
+        - title: AWS VPC Peering Gateway
           type: object
           properties:
             cidr_blocks
```

Reply:
@mheap here's a draft PR with what it looks like with this change - https://github.com/Kong/terraform-provider-konnect/pull/141/files#diff-a15f84a3853ee7ae0df5b199183360c02c5510286e2b1dbd424d8fbf6b70795f

Reply:
This looks right to me. Please apply the change